src/twisted/newsfragments/9006.feature

```diff
@@ -0,0 +1,2 @@
+twisted.trial.successResultOf, twisted.trial.failureResultOf, and
```

Comment:
these aren't actually the fqpns for these functions?

Reply:
The fqpns are in private modules; these are (I think) the public names.

Reply:
Like I don't think I want to say `twisted.trial._synctest._Assertions.successResultOf` here.